src/clj/rems/api/permissions.clj

```diff
@@ -14,11 +15,13 @@
     :tags ["permissions"]
 
     (GET "/:user" []
-      :summary "Returns user's permissions in ga4gh visa format. See https://github.com/ga4gh-duri/ga4gh-duri.github.io/"
+      :summary "Returns user's permissions in ga4gh visa format. Currently signed with fake key. See https://github.com/ga4gh-duri/ga4gh-duri.github.io/"
```

Comment:
should the config key be mentioned here?

Reply:
I would think no, it is not mentioned in the other experimental endpoint either that is enabled/disabled through config file.

Reply:
well at least add a "(EXPERIMENTAL)" or something